src/bootstrap/bin/rustc.rs

```diff
@@ -169,11 +173,6 @@ fn main() {
 
         // Set various options from config.toml to configure how we're building
         // code.
-        if env::var("RUSTC_DEBUGINFO") == Ok("true".to_string()) {
-            cmd.arg("-g");
```

Comment:
@petrochenkov: I'm no longer getting line information in stack traces (or anything other than `<unknown>`). Is it really not necessary to pass `-g`?